clang/test/CheckedCRewriter/testgenerator.py

```diff
@@ -484,7 +484,7 @@ def process_file(file, cname, cname2, proto, alltypes, new_defs, new_defs2, susp
     if alltypes: check = "CHECK_ALL"
 
     keywords = "int char struct double float".split(" ") 
-    ckeywords = "_Ptr _Array_ptr _Nt_array_ptr".split(" ")
+    ckeywords = "_Ptr _Array_ptr _Nt_array_ptr _Checked _Unchecked".split(" ")
```

Comment:
In your comment at the start of this PR you said you had backed out the changes that added checked regions, but it seems that the script still adds them?

Reply:
I have these added so when the checked regions issues get resolved, I can just run the script without changes. None of the tests have been updated with these keywords in them yet :)